src/dmd/dinterpret.d

```diff
@@ -88,31 +88,28 @@ public Expression ctfeInterpretForPragmaMsg(Expression e)
 
     // It's also OK for it to be a function declaration (happens only with
     // __traits(getOverloads))
-    if (e.op == TOK.variable && (cast(VarExp)e).var.isFuncDeclaration())
-    {
-        return e;
-    }
+    if (auto ve = e.isVarExp())
+        if (ve.var.isFuncDeclaration())
+        {
+            return e;
+        }
```

Comment:
I would rather but the braces on the outer `if` and remove them from the inner `if`.